spec/jobs/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -71,9 +74,13 @@
         VCR.use_cassette('evss/disability_compensation_form/submit_form_v2') do
           subject.perform_async(submission.id)
           expect_any_instance_of(EVSS::DisabilityCompensationForm::Metrics).to receive(:increment_retryable).once
-          expect(Form526JobStatus).to receive(:upsert).twice
```

Comment:
I don't have a good understanding of why, but this test line changed the behavior.  With it, Form526JobStatus was not saved.  🤷‍♀️ I think lines 79-83 cover this just as wel.